### PR TITLE
Added support for providing styles for labels separately

### PR DESCRIPTION
**Pull request by msaqlain**
Now you can pass props for labels separately for vertical and horizontal labels.

`propsForVerticalLabels` & `propsForHorizontalLabels`

### CODE REVIEW

**Comment by MuhammadAhmed1994** [comment]
Awesome (y)

**Comment by aboveyunhai** [comment]
> We can remove dist. Added just to point my package.json to my fork.

You can create dist folder for your own production deployment but you should't PR back to the main repo and remove `ignore dist`, unless there's a specific reason beneficial to the community or maintainer decided to include `dist` folder initially. Note that it will create tons of file and make repo hard to manage.  There are 48 files changed/add for one new feat. if you ever notices, but I think the important changes are just fewer files in `src` folder.

**Comment by msaqlain** [comment]
Any update on this @Hermanya ?

**Comment by Hermanya** [comment]
@msaqlain please revert the second commit

**Comment by msaqlain** [comment]
@Hermanya Reverted.

File: src/AbstractChart.tsx [hunk 1]
```diff
@@ -97,6 +97,30 @@ class AbstractChart<
...
   }
 
+  getPropsForVerticalLabels() {
+    const {
+      propsForVerticalLabels = {},
+      color,
+      labelColor = color
+    } = this.props.chartConfig;
+    return {
+      fill: labelColor(0.8),
```
Comment (Hermanya) [inline src/AbstractChart.tsx:107]: where does `labelColor(0.8)` come from? is this a new addition in this diff? why is it necessary?

**Comment by Hermanya** [comment]
the diff looks much better without the `dist` folder 👍

File: src/AbstractChart.tsx (hunk 1 above)
Comment (aboveyunhai) [reply src/AbstractChart.tsx:107]: > where does `labelColor(0.8)` come from? is this a new addition in this diff? why is it necessary?

https://github.com/indiespirit/react-native-chart-kit/blob/a499f7e212daf8e8f15286985e9007fcc0147818/src/AbstractChart.tsx#L95
Respect to the original `getPropsForLabels()`, looks good to me.

Comment (msaqlain) [reply src/AbstractChart.tsx:107]: Yes i havent changed the original props

**Comment by Hermanya** [comment]
thank you for your contribution, and I apologize for the delays. This is now available in 6.6.0

**Pull request merged**